src/Symfony/Component/Messenger/Command/FailedMessagesRetryCommand.php

```diff
@@ -59,6 +61,7 @@ protected function configure(): void
             ->setDefinition([
                 new InputArgument('id', InputArgument::IS_ARRAY, 'Specific message id(s) to retry'),
                 new InputOption('force', null, InputOption::VALUE_NONE, 'Force action without confirmation'),
+                new InputOption('failure-transport', null, InputOption::VALUE_OPTIONAL, 'Use a specific failure transport'),
```

Comment:
WDYT about just `--transport=`? `failure-transport` is more descriptive, but quite a lot to write!

Reply:
Perfect to me. It's clear the language, because in the end it's a transport.